### PR TITLE
doc: update tutorial

### PR DESCRIPTION
**Pull request by BethGriggs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,7 +233,7 @@ Add a Health Check endpoint to your Express.js application using the following s
    app.get("/healthz", (req, res) => res.status(200).json({ status: "ok" }));
    ```
 
- Add this line after the `app.use('/users', usersRouter);` line. This adds a `/healthz` endpoint to your application. As no liveness checks are registered, it will return as status code of 200 OK and a JSON payload of `{"status":"UP","checks":[]}`.
+ Add this line after the `app.get('/'...` line. This adds a `/healthz` endpoint to your application. As no liveness checks are registered, it will return as status code of 200 OK and a JSON payload of `{"status":"UP","checks":[]}`.
```
Comment (richardlau) [inline README.md:236]: Is this referring to `server.js`? The steps says `app.js` but I don't see that defined in the tutorial. And if it's the `server.js` defined earlier, this should be after the `app.use(helmet());` line?

Comment (BethGriggs) [reply README.md]: I think this is covered by other updates I still had locally (swapping `/healthz` to `/live` too)

File: README.md [hunk 2]
```diff
@@ -1,427 +1,629 @@
...
+   ```sh
+   mkdir express-app
+   cd express-app
+   ```
+
+2. Intialize your project with `npm` and install the Express.js module: 
+
+   ```sh
+   npm init --yes
+   npm install express
```
Comment (joesepi) [inline README.md:167]: Does it make sense to add `--save` to these dependencies will be saved?

**Review by joesepi — APPROVED** [review]
I have some suggestions and I've only read through it, I haven't run it (I have run the previous version though, so I'm familiar), but I approve. Thanks!

LGTM

File: README.md (hunk 2 above)
Comment (BethGriggs) [reply README.md:167]: Not in this case, as of npm@5+ installed modules are saved by default

Comment (joesepi) [reply README.md:166]: WAT! 🤯  How did I not know this?

**Review by joesepi — APPROVED** [review]

**Pull request merged**